src/Raven.Server/Rachis/FollowerAmbassador.cs

```diff
@@ -804,7 +807,15 @@ private long InitialNegotiationWithFollower()
                 } while (true);
             }
         }
-        
+
+        private int GetFollowerVersion(LogLengthNegotiationResponse llr)
+        {
+            var version = llr.CommandsVersion ?? 400_000;
+            if (version == 400)
```

Comment:
Maybe put some comment here that `400` was used in the past but we switched to higher range